yfinance/base.py

```diff
@@ -261,7 +261,7 @@ def cleanup(data):
 
         # sustainability
         d = {}
-        if 'esgScores' in data:
+        if data['esgScores']:
```

Comment:
This not only checks for the existence of the key. It also checks that it has a truthy value